mitodl/ocw-studio
# populated referencing content field for the website content

### What are the relevant tickets?
closes https://github.com/mitodl/hq/issues/5851

### Description (What does it do?)
This adds the references of the website contents that are using this WebsiteContent

### How can this be tested?
1. Open `ocw-studio`
2. Add a new page or edit an existing one
3. From resource picker, add a link for an external resource or page.
<img width="766" alt="Screenshot 2024-10-28 at 2 13 50 PM" src="https://github.com/user-attachments/assets/d98fe15c-e3bb-43c4-baca-27fa6139da2c">

4. Go to the `Admin` panel and check the `referencing_content` field for the selected resource. it should have the reference of the page where the link was added.
5. Type links in editor which would create External Resources with page reference (refer [here](https://github.com/mitodl/ocw-studio/pull/2279) for details on testing this feature)
